### PR TITLE
Add Web/API page types, part 19

### PR DESCRIPTION
**Pull request by wbamberg**
See  https://github.com/mdn/content/issues/16255. This is part 19 of a series of PRs to add page types to the Web/API documentation.

### CODE REVIEW

File: files/en-us/web/api/reportingobserveroptions/index.md [hunk 1]
```diff
@@ -1,6 +1,7 @@
 ---
 title: ReportingObserverOptions
 slug: Web/API/ReportingObserverOptions
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/reportingobserveroptions/index.md:4]: For the record, this is a dictionary.

File: files/en-us/web/api/resizeobserver/unobserve/index.md [hunk 2]
```diff
@@ -1,6 +1,7 @@
 ---
 title: ResizeObserver.unobserve()
 slug: Web/API/ResizeObserver/unobserve
+page-type: web-api-instance-property
```
Comment (teoli2003) [inline files/en-us/web/api/resizeobserver/unobserve/index.md:4]:
```suggestion
page-type: web-api-instance-method
```

File: files/en-us/web/api/response/error/index.md [hunk 3]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Response.error()
 slug: Web/API/Response/error
+page-type: web-api-static-method
```
Comment (teoli2003) [inline files/en-us/web/api/response/error/index.md:4]: Good catch!

File: files/en-us/web/api/response/redirect/index.md [hunk 4]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Response.redirect()
 slug: Web/API/Response/redirect
+page-type: web-api-instance-method
```
Comment (teoli2003) [inline files/en-us/web/api/response/redirect/index.md:4]:
```suggestion
page-type: web-api-static-method
```

Am I wrong?

File: files/en-us/web/api/response/json/index.md [hunk 5]
```diff
@@ -1,6 +1,7 @@
 ---
 title: Response.json()
 slug: Web/API/Response/json
+page-type: web-api-instance-method
```
Comment (teoli2003) [inline files/en-us/web/api/response/json/index.md:4]: If I read the spec correctly: 
```suggestion
page-type: web-api-static-method
```

File: files/en-us/web/api/rsahashedimportparams/index.md [hunk 6]
```diff
@@ -1,6 +1,7 @@
 ---
 title: RsaHashedImportParams
 slug: Web/API/RsaHashedImportParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/rsahashedimportparams/index.md:4]: For the record, it is a dictionary.

File: files/en-us/web/api/rsahashedkeygenparams/index.md [hunk 7]
```diff
@@ -1,6 +1,7 @@
 ---
 title: RsaHashedKeyGenParams
 slug: Web/API/RsaHashedKeyGenParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/rsahashedkeygenparams/index.md:4]: For the record, this is a dictionary.

File: files/en-us/web/api/rsaoaepparams/index.md [hunk 8]
```diff
@@ -1,6 +1,7 @@
 ---
 title: RsaOaepParams
 slug: Web/API/RsaOaepParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/rsaoaepparams/index.md:4]: For the record, this is a dictionary.

File: files/en-us/web/api/rsapssparams/index.md [hunk 9]
```diff
@@ -1,6 +1,7 @@
 ---
 title: RsaPssParams
 slug: Web/API/RsaPssParams
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/rsapssparams/index.md:4]: For the record, this is a dictionary.

File: files/en-us/web/api/rtcicecandidatepair/index.md [hunk 10]
```diff
@@ -1,6 +1,7 @@
 ---
 title: RTCIceCandidatePair
 slug: Web/API/RTCIceCandidatePair
+page-type: web-api-interface
```
Comment (teoli2003) [inline files/en-us/web/api/rtcicecandidatepair/index.md:4]: For the record, this is a dictionary.

**Review by teoli2003 — CHANGES_REQUESTED** [review]

File: files/en-us/web/api/response/redirect/index.md (hunk 4 above)
Comment (wbamberg) [reply files/en-us/web/api/response/redirect/index.md:4]: No, I think you are right: https://fetch.spec.whatwg.org/#response-class

File: files/en-us/web/api/response/json/index.md (hunk 5 above)
Comment (wbamberg) [reply files/en-us/web/api/response/json/index.md:4]: Oh this is fun. See https://github.com/openwebdocs/project/issues/104 and https://github.com/whatwg/fetch/pull/1392. I think we have a static and a non-static version of `Response.json()`. This page documents the non-static one, and we don't document the static one yet.

Comment (wbamberg) [reply files/en-us/web/api/response/json/index.md:4]: So this is correct, so far as it goes.

Comment (teoli2003) [reply files/en-us/web/api/response/json/index.md:4]: You are right, the non-static one is buried in a mixin and I didn't see it.

**Review by teoli2003 — APPROVED** [review]
👍

**Pull request merged**